ComfortZoneApp/app/src/main/java/com/example/comfortzone/Fragments/InputFragment.java

```diff
@@ -44,4 +55,31 @@ private void initViews(View view) {
         tvTime = view.findViewById(R.id.tvTime);
         tvCurrentTemp = view.findViewById(R.id.tvCurrentTemp);
     }
+
+    private void populateViews() {
+        tvCity.setText(weatherData.getCity());
+        tvCurrentTemp.setText(String.valueOf(weatherData.getTempData().getTemp()));
+    }
+
+    private void getWeatherClass() {
+        LocationUtil.getLastLocation(getActivity(), new GetLocationCallback() {
```

Comment:
love to see things coming together!

I would just call this `LocationCallback`, since you are referring to an object rather than a getter method